### ISSUE TITLE
Release a new version of broom.helpers

### ISSUE THREAD

**Issue by ddsjoberg** [larmarange/broom.helpers#112]
Hey hey @larmarange !! Hope you've been well!

I got a report about an issue with STAN regression models erring in the current version.
https://github.com/ddsjoberg/gtsummary/issues/852

If you don't have time to look at the details, I can do it this weekend.

``` r
packageVersion("broom.helpers")
#> [1] '1.2.1'
packageVersion("gtsummary")
#> [1] '1.3.7.9017'

fit_1 <- rstanarm::stan_glm(formula = mpg ~ cyl + wt,
                            data = mtcars,
                            family = gaussian(),
                            refresh = 0)

broom.mixed::tidy(fit_1)
#> # A tibble: 3 x 3
#>   term        estimate std.error
#>   <chr>          <dbl>     <dbl>
#> 1 (Intercept)    39.8      1.76 
#> 2 cyl            -1.51     0.439
#> 3 wt             -3.19     0.778
broom.helpers::tidy_plus_plus(fit_1, tidy_fun = broom.mixed::tidy)
#> Error in colSums(tcm * w): 'x' must be an array of at least two dimensions
gtsummary::tbl_regression(fit_1, tidy_fun = broom.mixed::tidy)
#> Error in colSums(tcm * w): 'x' must be an array of at least two dimensions
```

<sup>Created on 2021-04-08 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by ddsjoberg** [comment]
Oops, looks like you've already solved it in dev!

``` r
library(magrittr)

packageVersion("broom.helpers")
#> [1] '1.2.1.9000'
packageVersion("gtsummary")
#> [1] '1.3.7'

fit_1 <- rstanarm::stan_glm(formula = mpg ~ cyl + wt,
                            data = mtcars,
                            family = gaussian(),
                            refresh = 0)

broom.mixed::tidy(fit_1)
#> # A tibble: 3 x 3
#>   term        estimate std.error
#>   <chr>          <dbl>     <dbl>
#> 1 (Intercept)    39.7      1.71 
#> 2 cyl            -1.52     0.426
#> 3 wt             -3.20     0.765
broom.helpers::tidy_plus_plus(fit_1, tidy_fun = broom.mixed::tidy)
#> # A tibble: 2 x 15
#>   term  variable var_label var_class var_type   var_nlevels contrasts
#>   <chr> <chr>    <chr>     <chr>     <chr>            <int> <chr>    
#> 1 cyl   cyl      cyl       numeric   continuous          NA <NA>     
#> 2 wt    wt       wt        numeric   continuous          NA <NA>     
#> # ... with 8 more variables: contrasts_type <chr>, reference_row <lgl>,
#> #   label <chr>, n_obs <dbl>, estimate <dbl>, std.error <dbl>, conf.low <dbl>,
#> #   conf.high <dbl>
gtsummary::tbl_regression(fit_1, tidy_fun = broom.mixed::tidy) %>%
  gtsummary::as_tibble()
#> # A tibble: 2 x 3
#>   `**Characteristic**` `**Beta**` `**95% CI**`
#>   <chr>                <chr>      <chr>       
#> 1 cyl                  -1.5       -2.3, -0.65 
#> 2 wt                   -3.2       -4.8, -1.6
```

<sup>Created on 2021-04-08 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by ddsjoberg** [comment]
I am going to make a gtsummary release next week. Do you want to make another broom.helpers release, and I"ll make the new version the required minimum on the gtsummary side?

**Comment by larmarange** [comment]
Let me have a look this weekend about the ongoing PR on VGAM (to see if it can be inclue in this new release)

**Comment by larmarange** [comment]
will be fixed by #113
